### DISCUSSION TITLE
user-guide/plugins/easier-citation

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#161] [文档评论区]
# user-guide/plugins/easier-citation

Easier Citation for Zotero Todo

https://zotero-chinese.com/user-guide/plugins/easier-citation.html

**Comment by quan138898** [comment]
多重引用时，只删除一个如何操作？

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/faqs/word-addon.html#%E6%8F%92%E5%85%A5%E5%BC%95%E7%94%A8%E5%B8%B8%E8%A7%81%E7%9A%84%E6%93%8D%E4%BD%9C%E9%97%AE%E9%A2%98

两种模式下，第一种直接删除即可，第二种选中需要删除的后点 ← 按钮。

**Comment by iampearl918** [comment]
关闭word之后，在Zotero生成的临时文件夹中，变成了我自从下载使用Zotero以来的所有文献、甚至包括我这两年来删除的文献！
目前“我的文库”共1000多条文献，这个临时文件夹中有2000多条文献😂
这个插件是这样的吗？😂

**Reply by noneaddress** [reply]
我的现在也是这样的，显示的数量大于总数量

**Comment by JERRY-CLOIN** [comment]
我在mac上，添加插件之后，我的zotero 7打开应用功能之后，使用引用也不会在zotero引用列显示。请问如何解决？

**Reply by northword** [reply]
是完全没有「引用」列，还是这一列里面是空的？
是否打开了 word 文档并 refresh 了？

**Comment by RyanLai-Panda** [comment]
下载了最新版的easier citation，使用的是最新版zotero7，没有显示引用列，应该怎么办

**Comment by JERRY-CLOIN** [comment]
你如果是mac，可以关闭其他所有插件，然后退出zotero，重新打开；打开word点击refresh；然后在zotero里点一下其他的collect再点回来，就有了

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: RyanLai-Panda ***@***.***>
发送时间: Sunday, September 22, 2024 4:23:48 PM
收件人: zotero-chinese/wiki ***@***.***>
抄送: JERRY-CLOIN ***@***.***>; Comment ***@***.***>
主题: Re: [zotero-chinese/wiki] user-guide/plugins/easier-citation (Discussion #161)


下载了最新版的easier citation，使用的是最新版zotero7，没有显示引用列，应该怎么办

―

**Comment by Mia1423584160** [comment]
用的zotero7，下的也是7适配的插件，但是我完全找不到引用列在哪，右键那一列里完全没有“引用”

**Reply by wuzhaoqing2001** [reply]
你关闭Etheral style插件然后再打开试试看有没有引用，我这两个插件冲突了

**Reply by wuzhaoqing2001** [reply]
如果还不行你就只开这一个插件试试

**Comment by ROC-TON** [comment]
能不能修改快捷键？

**Comment by Lionelzq** [comment]
大神我想请教一下，为什么我在使用WPS删除一篇引用的文献时候，我先delete了所有正文的引注，然后再在zotero的库和临时文件夹中把文献给放进回收站，然后回到WPSrefresh，但是参考文献列表仍然有这篇文献，而且我删除了仍会在
